__tests__/app.test.js

```diff
@@ -44,3 +43,46 @@ describe("GET", () => {
       });
   });
 });
+
+describe("GET", () => {
+  it("should get 200 if respond with the message are all is well", () => {
+    return request(app)
+      .get("/api/reviews")
```

Comment:
this path will be a later endpoint so this will need changing in the coming tickets